src/util/focus-trap.ts

```diff
@@ -25,7 +25,7 @@ export function getFocusableBoundaryElements(element: HTMLElement): HTMLElement[
  * @param stopFocusTrap$ The observable stream. When completed the focus trap will clean up listeners
  * and free internal resources
  */
-export const ngbFocusTrap = (element: HTMLElement, stopFocusTrap$: Observable<any>) => {
+export const ngbFocusTrap = (element: HTMLElement, stopFocusTrap$: Observable<any>, refocusOnClick = false) => {
```

Comment:
could you please add `refocusOnClick` to jsdoc, just because two other arguments are there?

Reply:
done